backend/api/pkg/models/dbData.go

```diff
@@ -2,24 +2,36 @@ package models
 
 import "github.com/jinzhu/gorm"
 
-func initialiseTables(GDB *gorm.DB) {
+func initialiseTables(db *gorm.DB) {
 	for _, resource := range initResource {
-		GDB.Create(&resource)
+		db.Create(&resource)
 	}
 	for _, category := range initCategories {
-		GDB.Create(&category)
+		db.Create(&category)
 	}
 	for _, tag := range initTags {
-		GDB.Create(&tag)
+		db.Create(&tag)
 	}
 	for _, resourceTags := range initResourceTag {
-		GDB.Create(&resourceTags)
+		db.Create(&resourceTags)
 	}
 	for _, githubDetail := range initGithubDetail {
-		GDB.Create(&githubDetail)
+		db.Create(&githubDetail)
 	}
 	for _, resourceRawPath := range initResourceRawPath {
-		GDB.Create(&resourceRawPath)
+		db.Create(&resourceRawPath)
+	}
+	for _, resource := range initResource2 {
+		db.Create(&resource)
+	}
+	for _, githubDetail := range initGithubDetail2 {
```

Comment:
the need to use 1, 2, seems to me like a smell

Reply:
Updated !!